src/Microsoft.Health.Dicom.SqlServer/Registration/DicomServerBuilderSqlServerRegistrationExtensions.cs

```diff
@@ -104,6 +106,8 @@ public static class DicomServerBuilderSqlServerRegistrationExtensions
               .AsSelf()
               .AsImplementedInterfaces();
 
+            services.AddScopedDefault<SqlReindexStore>();
```

Comment:
Does this go to separate registrationExtension called by functions?

Reply:
yes, SqlReindexStore should only be used by Dicom functions, this line should be removed.